src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java

```diff
@@ -93,9 +102,19 @@ void findBySimpleProperty() {
 			vie = new Airport("airports::vie", "vie", "loww");
 			airportRepository.save(vie).block();
 			List<Airport> airports1 = airportRepository.findAllByIata("vie").collectList().block();
-			assertEquals(1,airports1.size());
+			assertEquals(1, airports1.size());
 			List<Airport> airports2 = airportRepository.findAllByIata("vie").collectList().block();
-			assertEquals(1,airports2.size());
+			assertEquals(1, airports2.size());
+			vie = airportRepository.save(vie).block();
+			List<Airport> airports = airportRepository.findAllByIata("vie").collectList().block();
+			assertEquals(1, airports.size());
+			System.out.println("findAllByIata(0): " + airports.get(0));
```

Comment:
looks like some debugging leftovers - I'd recommend to remove all those prints from tests since they make them very noisy in logfiles

Reply:
removed